options/internal/include/bits/ensure.h

```diff
@@ -21,6 +21,15 @@ void __ensure_warn(const char *assertion, const char *file, unsigned int line,
 #define MLIBC_MISSING_SYSDEP() __ensure_warn("Library function fails due to missing sysdep", \
 		__FILE__, __LINE__, __func__)
 
+#define MLIBC_CHECK_OR_ENOSYS(sysdep) ({ \
```

Comment:
As I mentioned in some random line, it's probably best to make this also not repeat `mlibc::sys_`.

Reply:
The downside of that is we won't be able to do `CHECK(mlibc::sys_foo & mlibc::sys_bar)`, and it's a little less obvious to the reader what is happening (+ less greppable). But maybe that's not a big deal and we can split it into separate lines